src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -104,6 +104,10 @@
         // Use on a card that remembers a memory, which is mechanic specific to the Wanderer character.
         @SpireEnum(name = "REMEMBER_MEMORY")
         public static AbstractCard.CardTags REMEMBER_MEMORY;
+
+        // Use on a card that performs metaupgrades, like Toil.
+        @SpireEnum(name = "METAUPGRADE")
```

Comment:
`PERSISTENT_POSITIVE_EFFECT` is the one to re-use (it already has the associated patches to remove them from temporary pools like skill potion)